python/lsst/ip/isr/isrTask.py

```diff
@@ -2105,7 +2130,9 @@ def maskAndInterpolateDefects(self, exposure, defectBaseList):
         defectBaseList : `List` of `Defects`
```

Comment:
Pedantic point but I think this is:
```
`list` of `~lsst.afw.image.DefectBase`
```
since `Defects` is going to be `~lsst.meas.algorithms.Defects` which is itself a collection of `lsst.meas.algorithms.Defect`.  In general it's better to use `Defects` right from the start if at all possible.

Reply:
Fixed to match the `maskAndInterpolate` method, and also fixed to remove interpolation mention in the not-interpolating method.